Sources/SwiftCurrent/SwiftCurrent.docc/Tutorials/Core.tutorial

```diff
@@ -110,9 +142,45 @@
                 `Any?` is broad enough to also include `AnyWorkflow.PassedArgs`. It's just a Swift way of saying "it doesn't matter what gets passed". `AnyWorkflow.PassedArgs` is more narrow, it means only certain `FlowRepresentable` types can pass data to this `FlowRepresentable.`
                 
                 @Justification(reaction: "Try again!") {
-                    Remember, `AnyWorkflow.PassedArgs` was deliberately built because an Optional wasn't enough to differentiate between "No data" and "Data, but it's nil". While convention may lead you to believe that `Any?` is more broad than `AnyWorkflow.PassedArgs` that's not true in this context.
+                    `AnyWorkflow.PassedArgs` was deliberately built because an Optional wasn't enough to differentiate between "No data" and "Data, but it's nil". While convention may lead you to believe that `Any?` is more broad than `AnyWorkflow.PassedArgs` that's not true in this context.
                 }
             }
-        }  
+        }
+        
+        @MultipleChoice {
+            What is the node's value type in the linked list?
```

Comment:
Was this meant to ask about nodes on a workflow? Cause for a LinkedList it's generic, that's how it's built. This whole multiple choice section doesn't quite click for me as written.

Reply:
Yeah, I was having trouble wording the question to make sense.  Essentially, when we talk about Workflow, I sometimes get confused at the many different levels of the layout (I tried drawing ASCII in the questions but it's unreliable).

If you look at what I've marked as the correct answer, _WorkflowItem is the item that you may overlook.  When you are looking at a node, the type for the value of the node is _WorkflowItem.  Node is not the correct answer because that's just the type of the node in a linked list, but not the value the node is holding/pointing to.

The importance of the question is that your Workflow is NOT a list of FR. It is also NOT a 
 list of FlowRepresentableMetadata.  But a _WorkflowItem holds both of these things.

Reply:
```suggestion
            What is the type used for the value of the node in the Workflow linked list?
```

Maybe this is slightly clearer?  Maybe it needs a drastic rewrite if this isn't any better.

Reply:
I think it gets fixed by splitting it into a question and a statement.
```suggestion
            A ``Workflow`` is a ``LinkedList``. What is the generic type of ``LinkedList.Node`` in a ``Workflow``?
```

Reply:
Additionally, I think there was confusion because your first answer isn't wrong??? It *is* a node. it's a `Node<_WorkflowItem>`